.github/workflows/rust.yml

```diff
@@ -0,0 +1,15 @@
+name: Rust
+
+on: [push]
```

Comment:
can you change this to `[push, pull_request]` please? this way the CI status will show up in this PR too

Reply:
> can you change this to `[push, pull_request]` please? this way the CI status will show up in this PR too

Done.